### ISSUE TITLE
one-var-declaration-per-line default is `initializations` not `always` as documented

### ISSUE THREAD

**Issue by Mardak** [eslint/eslint#6017]
**What version of ESLint are you using?**
2.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
`one-var-declaration-per-line: 2`

**What did you do? Please include the actual source code causing the issue.**
```js
var a, b;
```

**What did you expect to happen?**
`1:8  error  Expected variable declaration to be on a new line            one-var-declaration-per-line`

**What actually happened? Please include the actual, raw output from ESLint.**
no error

Docs say the default is `always`:
https://github.com/eslint/eslint/blame/master/docs/rules/one-var-declaration-per-line.md#L27

The code checks `var always = context.options[0] === "always";` so empty option results in `always` being `false`:
https://github.com/eslint/eslint/blob/master/lib/rules/one-var-declaration-per-line.js#L29

Probably safer to change the docs to say `initializations` is the default than changing the logic to make `always` actually be the default.

**Comment by Mardak** [comment]
@alberto Did you mean to make `always` the default or `initializations` for #1622 in #5016?

**Comment by alberto** [comment]
Thanks @Mardak for pointing this out and for the complete report.

I don't remember what I intended to do and we forgot to discuss that in the original issue :/.

I think changing the default to `always` would be a breaking change, so we can't do that and we should correct the docs instead to match the actual default. @nzakas agree?

**Comment by BYK** [comment]
@nzakas ping :)

**Comment by nzakas** [comment]
Agree. (Sorry, got buried in issues this week.)
